### PR TITLE
248/spanish button

### PR DESCRIPTION
**Pull request by cjhaddad**
#248  💅🏼 What issue does this fix?

This is a PR for adding a button to the navigation menu that allows the user to switch the website between english and spanish. It also adds a responsive menu that when below 768, you have a hamburger menu with language switch link and logout. 

## 🍂 Before You Submit

* [ ] Did you write tests?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Did you run `bundle exec rspec` from the root?
* [ ] Did you run `bundle exec rails rswag` from the root?
* [ ] Did you run `bundle exec rubocop` from the root?
* [ ] Did you run `yarn lint` in `/client`?
* [ ] Did you run `yarn test` in `/client`?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Data Migrations
* [ ] Schema Migrations
* [ ] Dependencies

### CODE REVIEW

File: client/src/_shared/__tests__/LoggedInLayout.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { LoggedInLayout } from '../LoggedInLayout'
+
+const doRender = overrideProps => {
+  const defaultProps = {}
+  return render(<LoggedInLayout {...defaultProps} {...overrideProps} />)
+}
+
+
+describe('<LoggedInLayout />', () => {
+  it('renders the CasesImport page', () => {
```
Comment (katelovescode) [inline client/src/_shared/__tests__/LoggedInLayout.test.js:13]:
```suggestion
  it('renders the LoggedInLayout wrapper', () => {
```

**Review by katelovescode — APPROVED** [review]
Looks great!  I pulled it down and tested it and it works beautifully.  I made one suggestion for test description, but ✅ looks good to me!

**Comment by PranavChaudhary1995** [comment]
Please unsubscribe me

File: client/src/_shared/LoggedInLayout.js [hunk 2]
```diff
@@ -12,7 +13,8 @@ const { useBreakpoint } = Grid
 
 export function LoggedInLayout({ children, title }) {
   const dispatch = useDispatch()
-  const { t } = useTranslation()
+  const { t, i18n } = useTranslation()
+  const [language, setLanguage] = useState(i18n.language)
```
Comment (arku) [inline client/src/_shared/LoggedInLayout.js:17]: Why do we need to hold the language state? Can't we use the state available from `react-i18next`?

File: client/src/_shared/LoggedInLayout.js [hunk 3]
```diff
@@ -21,6 +23,34 @@ export function LoggedInLayout({ children, title }) {
...
   }
 
+  const changeLanguage = lang => {
+    setLanguage(lang)
+    i18n.changeLanguage(lang)
+  }
+
+  const mobileMenu = (
+    <Menu>
+      {language === 'es' ? (
```
Comment (arku) [inline client/src/_shared/LoggedInLayout.js:33]: See https://github.com/pieforproviders/pieforproviders/pull/472/files#r511680452. We can use `i18n.language` here.

Comment (cjhaddad) [reply client/src/_shared/LoggedInLayout.js:33]: cool! good call.

**Pull request merged**